app/dashboard/helpers.py

```diff
@@ -67,7 +67,7 @@ def title(request):
     urlVal = URLValidator()
     try:
         urlVal(url)
-    except ValidationError, e:
+    except ValidationError as e:
```

Comment:
Total nit pick but if you aren't using any information from the Error Message you shouldn't even need the `as e` part.

Reply:
💯